backend/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ repositories {
 }
 
 dependencies {
-    implementation("com.google.protobuf:protobuf-kotlin:3.25.2")
+    implementation("com.google.protobuf:protobuf-kotlin:4.27.2")
```

Comment:
Remembering how I struggled with the protobuff setup, I'm scared to test this change 😄

Reply:
@seluianova I get that, would be super cool if you could still test this though please :pray:

Reply:
maybe would be helpful to specify the exact version of proto compiler in the dev instruction?